docs/ovs_communication.adoc

```diff
@@ -65,9 +85,31 @@ TBD - XiaoDong
 
 . when and who will initialize OVS? What are the initialization steps
 .. br-int, br-tun are created during init time, and recreated later in RPC loop if needed
-.. need to determine the minimal initialization steps
-. what happen when there are multiple physical NICs on the system, which NIC do we pick to hook up to br-tun, br-vlan, br-ex (if needed)?
+A: for the 1st questions:
+.. there are several ways to init ovs, one is start openvswith service directly or we could do the following
+a. start ovsdb-server, b. start vswitchd, use cmd to init as the following cmd:
+
+a. ovsdb-server --remote=punix:/usr/local/var/run/openvswitch/db.sock  \
+  --remote=db:Open_vSwitch,Open_vSwitch,manager_options \
+  --private-key=db:Open_vSwitch,SSL,private_key  \
+  --certificate=db:Open_vSwitch,SSL,certificate     \
+  --bootstrap-ca-cert=db:Open_vSwitch,SSL,ca_cert  \
+  --log-file=/var/log/openvswitch/ovs-vswitchd.log \
+  -vsyslog:dbg -vfile:dbg  --pidfile --detach
+ 
+b. ovs-vswitchd -v --pidfile --detach \
+ --log-file=/var/log/openvswitch/ovs-vswitchd.log \
+ -vconsole:err -vsyslog:info -vfile:info
+c. ovs-vsctl --no-wait init
+
+for the 2nd questions:
+a. create bridge (adding internal port)
```

Comment:
are we going to create the below, what is the exact command to create them?
br-int
br-tun
br-ex (for network node, or compute node with DVR)

Reply:
1. br-int ovs-vsctl add-br
2. br-tun ovs-vsctl add-port

Reply:
br-int : ovs-vsctl add-br
br-tun: ovs-vsctl add-port